README.md

```diff
@@ -51,6 +58,15 @@ Or minified:
 <script src="https://js.lisk.io/lisk-elements-1.0.0.min.js"></script>
 ```
 
+### Installation from source
+
```

Comment:
We need a bit of introduction. E.g.

"Our source code is hosted on GitHub. You can build the distribution yourself by cloning the repository, installing the relevant dependencies and running our build script as follows:"